libraries/core-react/src/components/Tabs/index.ts

```diff
@@ -7,16 +7,24 @@ import { TabPanel } from './TabPanel'
 type TabsCompoundProps = typeof BaseComponent & {
   Tab: typeof Tab
   TabList: typeof TabList
+  // Deprecated
   TabPanels: typeof TabPanels
   TabPanel: typeof TabPanel
+  // New
+  Panels: typeof TabPanels
+  Panel: typeof TabPanel
 }
 
 const Tabs = BaseComponent as TabsCompoundProps
 
 Tabs.Tab = Tab
+// Deprecated
 Tabs.TabList = TabList
```

Comment:
We skip `Tab.TabList`  -> `Tab.List` ?

Reply:
This one was hard 🙈  I did change it initially, but when I updated the storybook example even I was confused by the name List inside a Tabs 🤪  So I changed it back and added a comment about this in the commit message.